homeassistant/components/aidot/light.py

```diff
@@ -0,0 +1,112 @@
+"""Support for Aidot lights."""
+
+from typing import Any
+
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS,
+    ATTR_COLOR_TEMP_KELVIN,
+    ATTR_RGBW_COLOR,
+    ColorMode,
+    LightEntity,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import (
+    CONNECTION_NETWORK_MAC,
+    DeviceInfo,
+    format_mac,
+)
+from homeassistant.helpers.entity_platform import AddConfigEntryEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import AidotConfigEntry, AidotDeviceUpdateCoordinator
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: AidotConfigEntry,
+    async_add_entities: AddConfigEntryEntitiesCallback,
+) -> None:
+    """Set up Light."""
+    device_manager_coordinator = entry.runtime_data
+    async_add_entities(
+        AidotLight(hass, update_coordinator)
+        for update_coordinator in device_manager_coordinator.device_coordinators.values()
+    )
+
+
+class AidotLight(CoordinatorEntity[AidotDeviceUpdateCoordinator], LightEntity):
+    """Representation of a Aidot Wi-Fi Light."""
+
+    _attr_has_entity_name = True
+    _attr_name = None
+
+    def __init__(
+        self, hass: HomeAssistant, coordinator: AidotDeviceUpdateCoordinator
+    ) -> None:
+        """Initialize the light."""
+        super().__init__(coordinator)
+        self._attr_unique_id = coordinator.device_client.info.dev_id
+        self._attr_max_color_temp_kelvin = coordinator.device_client.info.cct_max
+        self._attr_min_color_temp_kelvin = coordinator.device_client.info.cct_min
+
+        modelId = coordinator.device_client.info.model_id
+        manufacturer = modelId.split(".")[0]
+        model = modelId[len(manufacturer) + 1 :]
```

Comment:
```suggestion
        model_id = coordinator.device_client.info.model_id
        manufacturer = model_id.split(".")[0]
        model = model_id[len(manufacturer) + 1 :]
```
use snakecase naming